### PR TITLE
[JA] Add `Help Centre`

### PR DESCRIPTION
**Pull request by suzuk12**
- [x] The changes are tested against the contribution checklist
- [x] (translations only) The changes are reviewed on GitHub by a fluent speaker

### CODE REVIEW

**Comment by suzuk12** [comment]
I checked the writing's on Windows 10.

File: wiki/Help_centre/ja.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
+---
+tags:
+  - help
+  - issue
+  - problem
+  - trouble
+  - missing
```
Comment (aceticke) [inline wiki/Help_centre/ja.md:7]: In addition to these, Japanese tags should be added, such as 問題 or サポート (used in the context of helping someone or providing support, not account support)

Comment (suzuk12) [reply wiki/Help_centre/ja.md:7]: We appreciate your input. The addition of the Japanese tags you mentioned, has been successfully completed.
```
  - ヘルプ
  - 課題
  - 問題
  - トラブル
  - サポート
```

Comment (aceticke) [reply wiki/Help_centre/ja.md:7]: いいね

**Comment by Sitoria** [comment]
out of context)

dont delete this on desc:

## Self-check

- [ ] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [ ] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

first one is `Self-Check`, see [this wiki page](https://osu.ppy.sh/wiki/ja/osu%21_wiki/Contribution_guide#t%E1%BB%B1-ki%E1%BB%83m-tra), check your md file and [x] 'd first one.

second one is for `This PR is reviewed by native person and approved.` ask other osu! JA users for grammer and sentence correctly - **Review**. if reviewer think its done, then reviewed user *approved* the PR. then you can [x] 'd it second box.

if second box is checked, then wiki maintainers will approved your translation and merged it to wiki folder. if not, it will merged 1 ~ 4 weeks later with `no_native_review` box + `stale merge` tag. no matter to wiki translations, but wiki translation quality is not guaranteed by other JA users.

hope you enjoy translating :>

**Comment by suzuk12** [comment]
> out of context)
> 
> dont delete this on desc:
> 
> ## Self-check
> * [ ]  The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
> * [ ]  _(translations only)_ The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)
> 
> first one is `Self-Check`, see [this wiki page](https://osu.ppy.sh/wiki/ja/osu%21_wiki/Contribution_guide#t%E1%BB%B1-ki%E1%BB%83m-tra), check your md file and [x] 'd first one.
> 
> second one is for `This PR is reviewed by native person and approved.` ask other osu! JA users for grammer and sentence correctly - **Review**. if reviewer think its done, then reviewed user _approved_ the PR. then you can [x] 'd it second box.
> 
> if second box is checked, then wiki maintainers will approved your translation and merged it to wiki folder. if not, it will merged 1 ~ 4 weeks later with `no_native_review` box + `stale merge` tag. no matter to wiki translations, but wiki translation quality is not guaranteed by other JA users.
> 
> hope you enjoy translating :>

Thank you for your response. I have confirmed the update.

**Review by TPGPL — APPROVED** [review]

**Pull request merged**